templates/shared/searchbottom.tmpl

```diff
@@ -10,3 +11,4 @@
 			{{end}}
 		</div>
 </div>
+{{end}}
```

Comment:
Again I have to ask to indent this.

Reply:
Again, I don't like unnecessary indentation, and it would conflict with "gt-" replacement. Last time I helped to fixed the conflicts in the `gt-` replacement PR , because that time it was unavoidable.  But this time, it could be avoided, so I prefer to keep it as-is.

```
{{if}
    <div></div>
{{end}}

{{if}
<div></div>
{{end}}
```


They all look good to me.

Reply:
I guess top-level is borderline okay to not indent.